lib/features/thread/presentation/widgets/suggestion_box_widget.dart

```diff
@@ -1,88 +1,152 @@
+              Text(
+                title,
+                maxLines: 1,
+                overflow: TextOverflow.ellipsis,
+                style: const TextStyle(
+                    fontSize: 13, color: AppColor.colorTextButtonHeaderThread),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
   }
 
   Widget build() {
-    return Material(
-      elevation: _elevation ?? 3,
-      color: Colors.white,
-      child: ConstrainedBox(
-        constraints: BoxConstraints(
-          maxHeight: _suggestionHeight ?? 60,
-        ),
-        child: ListView.builder(
-          shrinkWrap: true,
-          padding: EdgeInsets.zero,
-          itemCount: _listData.length,
-          itemBuilder: (BuildContext context, int index) => ListTile(
-            contentPadding: const EdgeInsets.symmetric(horizontal: 10),
-            onTap: () {
-              if (_onSelectedSuggestion != null) {
-                _onSelectedSuggestion!(_listData[index]);
-              }},
-            leading: IconButton(
-              color: AppColor.baseTextColor,
-              icon: SvgPicture.asset(_imagePaths.icSearch, color: AppColor.baseTextColor, fit: BoxFit.fill),
-              onPressed: () {
-                if (_onSelectedSuggestion != null) {
-                  _onSelectedSuggestion!(_listData[index]);
-                }
-              }),
-            title: Transform(
-              transform: Matrix4.translationValues(-12.0, 0.0, 0.0),
-              child: Row(
-                crossAxisAlignment: CrossAxisAlignment.center,
-                mainAxisAlignment: MainAxisAlignment.start,
+    return Container(
+      decoration: BoxDecoration(
+          boxShadow: [
+            BoxShadow(
+              color: Colors.grey.withOpacity(0.3),
+              spreadRadius: 0.5,
+              blurRadius: 2,
+              offset: const Offset(0, 0),
+            ),
+          ],
+          color: Colors.white,
+          borderRadius: const BorderRadius.all(
+            Radius.circular(20),
+          )),
+      child: SizedBox(
+        height: (_listData.length * 40 + 88),
+        child: Padding(
+          padding: const EdgeInsets.only(left: 12, top: 16),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Row(
                 children: [
-                  Text(
-                    '${AppLocalizations.of(_context).prefix_suggestion_search} ',
-                    style: const TextStyle(fontSize: 15, color: AppColor.baseTextColor),
-                  ),
-                  Expanded(child: Text(
-                    '"${_listData[index]}"',
-                    maxLines: 1,
-                    overflow: TextOverflow.ellipsis,
-                    style: const TextStyle(fontSize: 15, color: AppColor.colorNameEmail),
-                  ))
+                  suggestionFilterItem(
+                      'Has attachment', _imagePaths.icAttachmentsFilter),
```

Comment:
You should use `Applocalization` to be able to use multiple languages

Reply:
used localization